explainability-core/src/test/java/org/kie/trustyai/metrics/drift/fouriermmd/FourierMMDTest.java

```diff
@@ -41,6 +43,44 @@ protected void setup() throws Exception {
         testDF = readCSV(testDataSetFileName);
     }
```

Comment:
please be more specific with `throws`: you need to specify an `IOException` here, not a general`Exception`

Reply:
fixed